applications/workspaces/api/openapi.yaml

```diff
@@ -1018,13 +1049,13 @@ components:
           description: WorkspaceResource type
           $ref: "#/components/schemas/ResourceType"
         workspace_id:
-          description: The id of the Workspace this Workspace Resource belongs to
+          description: workspace_id
           type: integer
+          x-foreign-key: workspace.id
       required:
         - name
         - resource_type
         - location
-        - workspace_id
```

Comment:
workspace resource without a workspace id doesn't seem right to me

Reply:
When we create the resource templates in the frontend they don't have a workspace yet so this gets in the way